src/manifests/svc.yaml

```diff
@@ -0,0 +1,46 @@
+---
+# Source: https://github.com/ory/k8s/blob/master/helm/charts/kratos/templates/service-admin.yaml
+# helm chart v0.25.3
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ app_name }}-admin
+  labels:
+    app.kubernetes.io/component: {{ app_name }}-admin
+    app.kubernetes.io/name: {{ app_name }}
+    app.kubernetes.io/managed-by: juju
+    app.juju.is/created-by: {{ app_name }}
+    operator.juju.is/name: {{ app_name }}
+    operator.juju.is/target: application
+spec:
+  type: ClusterIP
+  ports:
+    - port: 80
+      targetPort: 4434
+      protocol: TCP
+      name: http
+  selector:
+    app.kubernetes.io/name: {{ app_name }}
+---
+# Source: https://github.com/ory/k8s/blob/master/helm/charts/kratos/templates/service-public.yaml
+# helm chart v0.25.3
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ app_name }}-public
+  labels:
+    app.kubernetes.io/component: {{ app_name }}-public
+    app.kubernetes.io/name: {{ app_name }}
+    app.kubernetes.io/managed-by: juju
+    app.juju.is/created-by: {{ app_name }}
+    operator.juju.is/name: {{ app_name }}
+    operator.juju.is/target: application
+spec:
+  type: ClusterIP
+  ports:
+    - port: 80
+      targetPort: 4433
+      protocol: TCP
+      name: http
+  selector:
+    app.kubernetes.io/name: {{ app_name }}
```

Comment:
I would drop this in favour of using the Kubernetes Service Patch lib if possible: https://charmhub.io/observability-libs/libraries/kubernetes_service_patch